packages/actor-query-operation-filter-sparqlee/lib/ActorQueryOperationFilterSparqlee.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ActorQueryOperation, ActorQueryOperationTypedMediated, Bindings,
+  IActorQueryOperationOutput,
+  IActorQueryOperationOutputBindings,
+  IActorQueryOperationTypedMediatedArgs,
+} from "@comunica/bus-query-operation";
+import { BindingsStream } from "@comunica/bus-query-operation";
+import { ActionContext, IActorTest } from "@comunica/core";
+import { Algebra } from "sparqlalgebrajs";
+import { AsyncEvaluator, ExpressionError } from "sparqlee";
+
+/**
+ * A comunica Filter Sparqlee Query Operation Actor.
+ */
+export class ActorQueryOperationFilterSparqlee extends ActorQueryOperationTypedMediated<Algebra.Filter> {
+
+  constructor(args: IActorQueryOperationTypedMediatedArgs) {
+    super(args, 'filter');
+  }
+
+  public async testOperation(pattern: Algebra.Filter, context: ActionContext): Promise<IActorTest> {
+    // Will throw error for unsupported operators
+    // Coerce to boolean to check for falsy values
+    return !!new AsyncEvaluator(pattern.expression);
```

Comment:
Is this an expensive operation? If so, perhaps we need to look into caching this somehow (not necessarily in this PR).

Reply:
It's not a simple if else check, but it's not performance that will matter when dealing with network data for example. Could prepend a very simple check first, but the full check is required to see if the operation is supported. Shouldn't matter too much in the end.